### PR TITLE
Scrap TinyMCE

### LINKED ISSUE #3108: Remove tinyMCE editor
This is a task tracker, following #3072 and [a public poll](https://helpers.pyxis.uberspace.de/display/ec054ce71458883f0a5973f868799590) by @annando, I set out to remove all traces of tinyMCE from friendica. However, this doesn't concern addons, these should be updated separately.

### PR DESCRIPTION
**Pull request by Hypolite**
Fixes #3108.

This is a major removal of code, I can't guarantee it's going to work first try, especially since it touches a lot of features I'm not regularly using like messages, events, etc...

I separated the PR into multiple commits for readability purpose.

Edit: Goodness, ````+537 −49,474```` ! 🙌

### CODE REVIEW

File: view/theme/frost-mobile/js/main.js [hunk 1]
```diff
@@ -445,13 +441,13 @@
 				$('#star-' + ident).addClass('hidden');
 				$('#unstar-' + ident).removeClass('hidden');
 			}
-			else {			
+			else {
```
Comment (annando) [inline view/theme/frost-mobile/js/main.js]: Could you remove that linefeed?

**Comment by Hypolite** [comment]
@annando Most of the line removal comes from the lib itself plus the TinyMCE files customized for Friendica. The first commit dealing with ````library```` account for 290 deleted files and 47,281 lines removed. The rest is mostly removing TinyMCE initialization.

**Comment by annando** [comment]
For the next version (3.5.2) we should have a look at unneeded library files. I think this could be a job that @Quix0r would like.

File: view/theme/frost-mobile/js/main.js (hunk 1 above)
Comment (Quix0r) [inline view/theme/frost-mobile/js/main.js]: He means `} else {`. :-)

**Comment by Quix0r** [comment]
To kill white-spaces automatically, there are e.g. shell scripts around, maybe also a `sed` command will do it. Then you don't need to look for them manually.

**Comment by rabuzarus** [comment]
Wow. Good work. 

> For the next version (3.5.2) we should have a look at unneeded library files.

Yes, and we should have a look if we could update some of the libraries we use

**Comment by tobiasd** [comment]
@annando has this one lf remova (see above)l, otherwise I'm fine with it as well and would merge it.

**Comment by Quix0r** [comment]
I had already checked all external libraries and some have newer (stable) versions. Please check my commits (maybe one day that may happen that we use sub modules, would be really awsome):

https://github.com/Quix0r/friendica/commits/rewrites/git-submodules

**Comment by annando** [comment]
@tobiasd You could always merge it. This single line is no showstopper.

**Pull request merged**